packages/replay-next/src/contexts/LayoutContext.tsx

```diff
@@ -0,0 +1,10 @@
+import { createContext } from "react";
+
+export type LayoutContextType = {
+  // whether console and sources can be shown at the same time
+  canShowConsoleAndSources: boolean;
```

Comment:
tiny nit: I wonder if `sourceViewerVisible` might be a clearer name? The only thing that uses this _is the Console_ view code, so it's kind of obvious that it's visible.